src/Http/Controllers/StaticController.php

```diff
@@ -11,6 +11,9 @@
 
 class StaticController extends Controller
 {
+    private const FILE_EXTENSION = [
```

Comment:
Oh, come on, this constant name doesn't tell anything about its content. Please make it more verbose.